doc/README.md

```diff
@@ -1,5 +1,24 @@
 # Composite Setup Guide
 
+## Tools nessesary for Composite
+```
+sudo apt-get -y install bc  
+sudo apt-get -y install gcc-multilib  
```

Comment:
We don't beed `multilib` for 32bit. should we note that?

Reply:
Doesn't really hurt to include it ;-)